clustermesh/certs.go

```diff
@@ -14,7 +14,7 @@ import (
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 
 	"github.com/cilium/cilium-cli/defaults"
-	"github.com/cilium/cilium-cli/internal/k8s"
+	"github.com/cilium/cilium-cli/pkg/k8s"
```

Comment:
I'd probably move the package to `github.com/cilium/cilium-cli/k8s`, i.e. without the `pkg/` part. All other public packages are at the top level already.

Reply:
updated!